subgraphs/compound-v2/src/mappings/comptroller.ts

```diff
@@ -0,0 +1,50 @@
+import { MarketEntered, MarketListed, NewCollateralFactor, NewLiquidationIncentive, NewMaxAssets, NewPriceOracle } from "../../generated/Comptroller/Comptroller"
+import { Protocol } from "../../generated/schema"
+import { CToken } from "../../generated/templates"
+import { getOrCreateAccount } from "../helpers/account"
+import { mantissaFactorBD, unitrollerAddress } from "../helpers/generic"
+import { getOrCreateMarket } from "../helpers/market"
+import { getOrCreateProtocol } from "../helpers/protocol"
+
+export function handleMarketListed(event: MarketListed): void {
+    // Dynamically index all new listed tokens
+    CToken.create(event.params.cToken)
+    // Create the market for this token, since it's now been listed.
+    let market = getOrCreateMarket(event.params.cToken.toHexString())
+    market.save()
+}
+
+
+export function handleMarketEntered(event: MarketEntered): void {
+    let market = getOrCreateMarket(event.params.cToken.toHexString())
+    let account = getOrCreateAccount(event.params.account.toHexString())
+    let protocol = getOrCreateProtocol(market.protocol)
+}
+
+export function handleNewPriceOracle(event: NewPriceOracle): void {
+    let protocol = getOrCreateProtocol(unitrollerAddress)
```

Comment:
`unitrollerAddress` doesn't need to be used anywhre here. you can just use `event.address` 

this probably will allow you to delete the hardcoded value of `unitrolleraddress`